ttjc@piyushverma001/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Piyush Verma
+type: "participant"
+name: "Piyush Verma"
+portfolio: "https://piyushverma.netlify.com"
+github-link: "https://github.com/piyushverma001"
+linkedin-link: "https://www.linkedin.com/in/piyush-verma-258800163/"
+looking-for: "internship"
+description: "Experienced at AI/ML and now  learning web development"
+---
+
+Hi! I am a Piyush Verma let me walk you through my intro, projects and blogs.
```

Comment:
Hey! @piyushverma001 a small mistake here:
* I am **a** Piyush ...

Reply:
I made the required changes. Please accept the request